README.md

```diff
@@ -43,8 +43,12 @@ dependencies {
 }
 ```
 
+If using Android Studio, you may need to go `File -> Invalidate Caches -> Restart` in order for the IDE to recognise MultiK.
```

Comment:
If it's true for Android Studio then I believe it would be true for any Intellij IDEA not just Android Studio.  
What do you think?

Reply:
Probably, can't confirm as I've only used Android Studio

Reply:
The steps you described are available on any Intellij IDEA. In fact I believe Android Studio was developed with the same IDEA base that is found in every intellij IDEA (it was built on top).

Might be beneficial to be more broad in the description or to mention "Android Studio or Intellij IDEA"